src/app/home/events/events.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container section center-align">
+  <div class="row center-align container">
```

Comment:
You are using container and center-align classes twice, these classes should only be applied for the div containing all the section.